api/src/main/java/org/pmiops/workbench/initialcredits/InitialCreditsService.java

```diff
@@ -341,19 +341,34 @@ public void extendInitialCreditsExpiration(DbUser user) {
     DbUserInitialCreditsExpiration userInitialCreditsExpiration =
         user.getUserInitialCreditsExpiration();
     if (userInitialCreditsExpiration == null) {
-      throw new WorkbenchException("User does not have initial credits expiration set.");
+      throw new WorkbenchException(
+          "User does not have initial credits expiration set, so they cannot extend their expiration date.");
```

Comment:
can u add a comment sth like `this should never happen after "migration"`

Reply:
Migration will handle the case where users already have access, but there is still the case that this endpoint is called for a user who does not yet have RT access and therefore no expiration record. There is no way to to call this endpoint in the UI for users without RT access, but since the api could be called directly, I think that this exception will be worth keeping even after migration. I added a comment.